### PR TITLE
Reinstall the pip.exe if it does not exist at the Python34 path

### LINKED ISSUE #154: Kolibri Installer failed to install
During the Windows installation there is a possibility that the `pip.exe` did not install in Python or it was deleted by an antivirus. To fix this validate the `pip.exe` path if it exist before installing the Kolibri.

The  deleted `pip.exe` will not reinstall at the old Python34 path installation, unless manually uninstall the Python34 at the control panel  and delete the `C:\Python34'(if this folder is still exist). before installing the Python34 again.

### PR DESCRIPTION
**Pull request by mrpau-richard**
Fixes #154

I added the bundled pip whl package to the installer. This will fix the issue of `pip.exe not found` during the Kolibri installation. 

To test this PR delete the `pip.exe` located at the `c:\Python34\Scripts` and install this  [installer](https://drive.google.com/open?id=1moQgihLIz8Lhp80rJyyl106t0EXMyPhz)

### CODE REVIEW

File: src/installer-source/KolibriSetupScript.iss [hunk 1]
```diff
@@ -495,34 +478,62 @@ end;
...
+
+function FailedPipNotFound() : String;
+begin
+    // MsgBox('FailedPipNotFound - ' + GetExceptionMessage, mbInformation, mb_Ok);
+    // Log('Exception: ' + GetExceptionMessage);
+    CustomizeMsgbox('File: ' + ExpandConstant('{sd}') + DEFAULT_PIP_PATH + CustomMessage('FileNotFound'));
+    RemoveOldInstallation(ExpandConstant('{app}'));
+    forceCancel := True
+    ExitProcess(1);
+end;
```
Comment (indirectlylit) [inline src/installer-source/KolibriSetupScript.iss]: do you know why the git diff on this file is so large? The text looks unchanged.

If there was some hidden change in endlines or whitespace, it would be better to do that in a separate commit to help see what has actually changed

Comment (benjaoming) [reply src/installer-source/KolibriSetupScript.iss]: Previously, there were 2 linebreaks (line 499 and 500), now there's one.. but yeah that's not a very big difference :)

Comment (benjaoming) [reply src/installer-source/KolibriSetupScript.iss]: And `GetDefaultPip` (below) is new..

Comment (indirectlylit) [reply src/installer-source/KolibriSetupScript.iss]: that wasn't quite what I was asking @benjaoming. I'm saying that the diff is showing changes even though the text is the same

Comment (mrpau-richard) [reply src/installer-source/KolibriSetupScript.iss]: > do you know why the git diff on this file is so large? The text looks unchanged.

I think its because of the white space, I try to restore it.

**Comment by cpauya** [comment]
@mrpau-richard Please fix the whitespace-related issue in a separate commit, I saw some inconsistent indentions in the [KolibriSetupScript.iss](https://github.com/learningequality/kolibri-installer-windows/blob/2bcf229afb5a34fc70ef47910efb11c42069ec81/src/installer-source/KolibriSetupScript.iss) and [python-exe.bat](https://github.com/learningequality/kolibri-installer-windows/blob/2bcf229afb5a34fc70ef47910efb11c42069ec81/src/python-setup/python-exe.bat) files.

Then you can integrate your fixes/changes in another commit to make them "atomic".

**Comment by benjaoming** [comment]
I'm new to this, but it seems clever: https://dev.to/koffeinfrei/the-git-fixup-workflow-386d

**Comment by cpauya** [comment]
I will test this on a Windows 7 VM and try if I can reproduce the "missing pip.exe" issue too.

**Comment by benjaoming** [comment]
If `pip.exe` does not exist because it was quarantined by anti-virus, will it help to re-add it?

**Comment by mrpau-richard** [comment]
>If pip.exe does not exist because it was quarantined by anti-virus, will it help to re-add it?

I did not test this scenario @benjaoming, I'll try to test it.

**Comment by mrpau-richard** [comment]
> If pip.exe does not exist because it was quarantined by anti-virus, will it help to re-add it?

@benjaoming I tested this using Avast antivirus, and yes! the pip will be re-add after I reinstall the Windows installer.

File: src/installer-source/KolibriSetupScript.iss [hunk 2]
```diff
@@ -441,6 +441,39 @@ begin
             RemoveOldInstallation(targetPath);
         end;
     end;  
+end;
+
+{ Used in GetPipPath below }
+const
+    DEFAULT_PIP_PATH = '\Python34\Scripts\pip.exe';
+{ Returns the path of pip.exe on the system. }
+{ Tries several different locations before prompting user. }
```
[the remark this answers is not in the document]
Comment (mrpau-richard) [reply src/installer-source/KolibriSetupScript.iss]: Yeah, that comment is not needed anymore I will remove it.

**Comment by indirectlylit** [comment]
thanks guys - if testing checks out, feel free to merge and we can cut a new release

**Comment by cpauya** [comment]
Tested on a Windows 10 VM:
1. Installed Kolibri v0.13.2-beta1
1. Stopped Kolibri and deleted `c:\Python34\Scripts\pip*.exe` files
1. Kolibri failed to start already
1. Installed the installer mentioned in this PR
1. Kolibri started okay with `pip*.exe` files in place.

Thanks @mrpau-richard - will merge now.

**Comment by indirectlylit** [comment]
Awesome. To follow up on all this, let's make sure to add the issue and PR to the 1.3.6 milestone and continue to close out the remaining items or triage them into the next release

then we can get kolibri 0.13.2 upgraded to this in time for release (cc @jonboiser, @lyw07) 

Created https://github.com/learningequality/kolibri/issues/6700 to track
